src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/IssueNavigatorPage.kt

```diff
@@ -38,14 +38,22 @@ class IssueNavigatorPage(
         return this
     }
 
-    fun getIssueKeys(): Set<String> {
+    private fun getIssueKeys(cssSelector: String): Set<String> {
         val issueKeys: List<String> = JavaScriptUtils.executeScript(driver,
-            "return Array.from(document.getElementsByClassName('issue-link-key'), i => i.innerText.trim())"
+            "return Array.from(document.getElementsByClassName('${cssSelector}'), i => i.innerText.trim())"
         )
 
         return HashSet(issueKeys)
     }
 
+    fun getIssueKeys(): Set<String> {
+        return getIssueKeys("issue-link-key")
+    }
+
+    fun getIssueKeysCloud(): Set<String> {
```

Comment:
Can Cloud-specific behavior be kept in the [inlined CloudIssueNavigator](https://github.com/atlassian/jces-1209/blob/b036176c7c568f0e1f8e75e265c9008244c1a1a6/custom-vu/src/main/kotlin/jces1209/vu/page/issuenavigator/CloudIssueNavigator.kt)?

Reply:
@mzyromski-atlassian I don't have a TravisCI account, will the lack of this block the potential merge?